puente-analytics-service/lambdas/etl/env.example.json

```diff
@@ -0,0 +1,14 @@
+{
+  "AnalyticsLambdaFunctionETL": {
+    "AWS_ACCESS_KEY_ID": "",
+    "AWS_SECRET_ACCESS_KEY": "",
+    "AWS_REGION": "",
+    "PG_HOST": "fakeinstance-us-east-1-dev-1.fakeid.us-east-1.rds.amazonaws.com",
+    "PG_PORT": "5432",
+    "PG_DATABASE": "dbname",
+    "PG_USERNAME": "username",
+    "PG_PASSWORD": "password",
+    "S3_BUCKET_NAME": "s3-bucket-with-csvs",
+    "S3_OBJECT_KEY": "s3-bucket-key"
+  }
+}
```

Comment:
@awalsh272 you'll create a copy of this, and it'll be called `env.json` This will replace the environment variables inside the cloudformation template